File: app/src/j/java/com/testspace/CurrentExperiment.java
package com.testspace;

import androidx.annotation.LayoutRes;

import com.madlab.core.Experiment;

/**
 * @see Experiment
 */
public class CurrentExperiment extends Experiment {
    CurrentExperiment(MainActivity a) {
        super(a);
        a.addTriggers(
                /* add your triggers */
        );
    }

    @Override @LayoutRes
    public int getExperimentLayout() {
        return R.layout.basic_layout;
    }
}

File: app/src/main/java/com/testspace/CurrentExperiment.kt
package com.testspace

import androidx.annotation.LayoutRes
import com.testspace.core.Experiment
import com.testspace.core.ExperimentActivity

/**
 * @see Experiment
 */
class CurrentExperiment(a: ExperimentActivity) : Experiment(a) {
    init {
        a.addTriggers(
                /* add triggers here */
        )
    }

    @LayoutRes
    override fun getExperimentLayout() = R.layout.basic_layout
}